include/c2pa.hpp

```diff
@@ -115,4 +140,616 @@ namespace C2pa
         c2pa_release_string(result);
         return;
     }
+
+    // IStream Class wrapper for CStream
+    class CppIStream : public CStream
```

Comment:
Code format consistency: Braces on same line or not? In the previous file, they were on same line (which I prefer).

Reply:
Broader meta question: Is there a code formatter akin to `rustfmt` that can be applied here? I find consistency in formatting helps to avoid distractions when reading.

Reply:
I'll run this through a formatter in vscode. We can look into automating that later.